### ISSUE TITLE
lighthouse: Performance

### ISSUE THREAD

**Issue by andrejusk** [andrejusk/andrejusk.github.io#12]
Defer unused CSS for 0.15s increase.

Thought it's already deferred?

URL | Size (KB) | Potential Savings (KB)
-- | -- | --
…css/bootstrap.min.css(cdnjs.cloudflare.com) | 20 KB | 20 KB

**Comment by andrejusk** [comment]
consistently 100/100 after #19, fonts fade in which makes it looks nice as well, contenful logo and header displays before fade

could replace text with bars while loading, somehow compress all svgs into a texturemap
